scripts/pagerduty.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+# Pagerduty constants
+LOGFILE="/home/tmp_log/log-20190628.153354/*.log"
+LOG=$(tail -n 1000 $LOGFILE)
+SHARD_NUM=$(echo "$LOG" | grep -oE -m 1 "Shard\":.?" | rev | cut -c 1)
+IP=$(dig -4 @resolver1.opendns.com ANY myip.opendns.com +short)
+KEY="5d11f6173899423689b6fc98691930de"
+TRIGGER="\
+{\
+  \"routing_key\": \"$KEY\",\
+  \"event_action\": \"trigger\",\
+  \"dedup_key\": \"$IP\",\
+  \"payload\": {\
+    \"summary\": \"Node NOT validating: Shard-$SHARD_NUM: $IP\",\
+    \"source\": \"$IP\",\
+    \"severity\": \"critical\"\
+  }\
+}"
+RESOLVE="\
+{\
+  \"routing_key\": \"$KEY\",\
```

Comment:
(Nit) auth materials such as this usually belongs to the HTTP request header in REST APIs.  Consider passing this as a header (`curl -H "X-Routing-Key: $KEY" ...`).